Sources/Sentry/SentryCrashWrapper.m

```diff
@@ -212,6 +219,25 @@ - (void)enrichScope:(SentryScope *)scope
     [appData setValue:systemInfo[@"buildType"] forKey:@"build_type"];
 
     [scope setContextValue:appData forKey:@"app"];
+
+    // Runtime
+    NSMutableDictionary *runtimeContext = [NSMutableDictionary new];
+
+    if (@available(iOS 14.0, macOS 11.0, watchOS 7.0, tvOS 14.0, *)) {
+        if (SentryDependencyContainer.sharedInstance.processInfoWrapper.isiOSAppOnMac) {
```

Comment:
I know this isn't ideal. I plan to open a follow-up PR for getting rid of the `SentryDependencyContainer.sharedInstance` and the  `SentryCrashWrapper.sharedInstance`. We already access the `SentryDependencyContainer.sharedInstance` a few times in this class and injecting all the dependencies via the initializer touches multiple files.

Reply:
Thanks for the comment.